modules/app/read_write.py

```diff
@@ -15,15 +16,19 @@ def __init__(self) -> None:
         self.textDir = self.dir.joinpath(self.settings.txt_subdir)
         self.pdfDir = self.dir.joinpath(self.settings.pdf_subdir)
 
+    class FilesDict(TypedDict):
+        filename: str
+        contents: bytes
+
     def getFiles(self, path: Path, count_only: bool = False) -> list:
         """Get all files in a certain directory."""
-        files = []
+        files: list[ReadWrite.FilesDict] = []
 
         if any(path.glob("*")):
             for file in path.glob("*"):
                 if file.is_file():
                     # Do not read the file if count_only is true
-                    contents = None if count_only else file.read_bytes()
+                    contents = "".encode() if count_only else file.read_bytes()
```

Comment:
Using ```python b''``` would've also work.

Reply:
That is certainly more concise, indeed; I’ve changed it to `b""`.